src/grt/grt-vhpi.ads

```diff
@@ -392,13 +386,15 @@ package Grt.Vhpi is
       VhpiSizeConstraint
      );
    pragma Convention (C, VhpiPutValueModeT);
+   for VhpiPutValueModeT'Size use Integer'Size;
```

Comment:
You don't need to specify the size of enumeration types.  This could contradict/override the pragma convention just above.

Reply:
At first I didn't specify the size, but compilation failed with this warning:
```
grt-vhpi.adb:648:07: warning: types for unchecked conversion have different sizes
grt-vhpi.adb:648:07: warning: size of "Vhpiputvaluemodet" is 3, size of "Integer" is 32
grt-vhpi.adb:648:07: warning: source will be extended with 29 high order zero bits
```

Is there a better approach for getting the invalid values for Mode that are passed from C?